specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2021-12-01/Marketplace.json

```diff
@@ -0,0 +1,2894 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StopSellOffersPlansNotificationsListProperties"
+          }
+        }
+      }
+    },
+    "StopSellOffersPlansNotificationsListProperties": {
+      "description": "List of stop sell offers and plans notifications.",
+      "type": "object",
+      "properties": {
+        "offerId": {
+          "description": "Gets offer id",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "Gets offer display name",
+          "type": "string",
+          "readOnly": true
+        },
+        "isEntire": {
+          "description": "Gets a value indicating whether entire offer is in stop sell or only few of its plans",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "messageCode": {
+          "description": "Gets the notification message id",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        },
+        "icon": {
+          "description": "Gets the icon url",
+          "type": "string",
+          "readOnly": true
+        },
+        "plans": {
+          "description": "Gets removed plans notifications",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PlanNotificationDetails"
+          },
+          "readOnly": true
+        },
+        "publicContext": {
+          "description": "Gets if the offer has public plans",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "subscriptionsContext": {
+          "description": "Gets the subscriptions context for private plans",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "StopSellPayload": {
+      "description": "Private plans subscriptions",
+      "type": "object",
+      "properties": {
+        "subscriptions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SubscriptionsResponse": {
+      "description": "Subscription list operation response.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Subscription"
+          },
+          "description": "An array of subscriptions.",
+          "readOnly": true
+        },
+        "skipToken": {
+          "type": "string",
+          "description": "The skip token to get the next page.",
+          "readOnly": true
+        },
+        "count": {
+          "description": "Gets the number of subscriptions on a page",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        }
+      }
+    },
+    "Subscription": {
+      "description": "Subscription information.",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

If this remains as part of a GET response it requires id, name, and type properties. Other properties (displayName, state, etc.) go into the property bag.

Reply:
Changes to post